python/utils/worker.py

```diff
@@ -173,8 +177,12 @@ def producer(
                     "channel_recv_latency_in_secs": str(
                         format(perf_counter() - last_insertion_time, ".8f")
                     ),
+                    "duration_in_secs": str(format(perf_counter() - start_time, ".8f")),
+                    "step": "1",
+                    "service_type": PROCESSOR_SERVICE_TYPE,
                 },
             )
+            q.put((chain_id, response.transactions))
```

Comment:
Is it simple to get `size_in_bytes` from response? If so, we can add it to the logs too

Reply:
done.

Reply:
done~